examples/src/main/java/io/servicefabric/examples/gossip/ClusterNodeB.java

```diff
@@ -16,14 +16,11 @@
 public class ClusterNodeB {
 
 	public static void main(String[] args) {
-		// Register data types (used for serialization)
-		TransportTypeRegistry.getInstance().registerType("hello/world", Greetings.class);
-		
 		// start cluster node that listen on port 3001 and point to node A as seed node
 		ICluster clusterB =  Cluster.newInstance(3001,"localhost:3000").join();
 		
 		// spread gossip
-		clusterB.gossip().spread("hello/world", new Greetings("Greetings from ClusterMember B"));
+		clusterB.gossip().spread(new Message(new Greetings("Greetings from ClusterMember B")));
```

Comment:
you can have also headers... we can provide additional helper method which will accept data and automatically create message without headers, but this is sugar - I didn't focus here on sugar just core part (this PR already too big )